src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -400,6 +400,8 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ngraph::Func
         {opset4::GreaterEqual::get_type_info_static(), fuse_type_to_binary_comparision<opset4::GreaterEqual>},
         {opset4::Less::get_type_info_static(), fuse_type_to_binary_comparision<opset4::Less>},
         {opset4::LessEqual::get_type_info_static(), fuse_type_to_binary_comparision<opset4::LessEqual>},
+        {opset10::IsFinite::get_type_info_static(), fuse_type_to_binary_comparision<opset10::IsFinite>},
+        {ov::opset10::IsNaN::get_type_info_static(), fuse_type_to_binary_comparision<ov::opset10::IsNaN>},
```

Comment:
I bet we also need to cover opset10::IsInf in the list

Reply:
Updated.